### ISSUE TITLE
Extension not appearing on dashboard

### ISSUE THREAD

**Issue by thebaht** [R0tenur/visualization#27]
## Expected Behavior
" _Show database schema Double tap your db and press "Schema Visualization" on the dashboard._ "

## Current Behavior
The extension doesn't show up on my dashboard, so I can't press "Schema Visualization" to show the schema

## Context (Environment)
Azure Data Studio
Version: 1.24.0 (system setup)
Commit: eb3c0f5d9df86b9490606aab7fffaa253b0aca4d
Date: 2020-11-10T23:38:09.705Z
VS Code: 1.48.0
Electron: 9.3.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19042

## Detailed Description

![Missing extension](https://i.imgur.com/1o30RPY.png)

**Comment by R0tenur** [comment]
Is this for sql-server?

**Comment by thebaht** [comment]
it's postgres

**Comment by R0tenur** [comment]
I've not manager to make it work for postgres yet.
It's not about grabbing the table data, but make the extention to appear. Have ypu seen other extentions where this works? If so I would like to check them out
 Related to #9

**Comment by thebaht** [comment]
No, I haven't seen another extension where it works yet. I only started using azure data studio less than a week ago

**Comment by DarkFM** [comment]
I also get the same issue when connecting to a local SQL Server Instance running in a Linux Docker Container
![image](https://user-images.githubusercontent.com/22402215/109229388-12324f80-7778-11eb-8a15-be0d8550a292.png)

**Comment by manishnextsteps** [comment]
I am experiencing the same issue while connecting to a local SQL Server Instance

**Comment by manishnextsteps** [comment]
Linux. Azure data studio version: 1.21.0

**Comment by alexyca** [comment]
I had a similar issus in a database in which I had custom insight. Deleting custom widget insight make it appears.

**Comment by anujmca** [comment]
I am also facing exactly same issue, any solution found so far?
